### ISSUE TITLE
v3.wttr.in bugs

### ISSUE THREAD

**Issue by chubin** [chubin/wttr.in#540]
v3.wttr.in works, but it is the first, minimal viable implementation,
and it has a lot of different bugs. All known bugs and feature requests should be gathered here.

![Sakha](https://user-images.githubusercontent.com/3875145/97636968-279da400-1a3a-11eb-87a5-4744e9bb4fc7.png)

Bugs:

* Corner cases (Alaska, Arctics, Antartica, Equator)

Feature requests:

* transparent background, see https://twitter.com/JakubUrbanec/status/1321926264844918787/photo/1

**Comment by claenn** [comment]
Would be nice to fix the emojis width for @iSH_app

![IMG_1179](https://user-images.githubusercontent.com/8040019/97638790-95979a80-1a3d-11eb-8a52-4b004cf7b81e.png)

**Comment by chubin** [comment]
@claenn It is #345

**Comment by mcepl** [comment]
https://v3.wttr.in/Czechia shows
![Czechia](https://user-images.githubusercontent.com/198999/109788579-bcdebe00-7c0f-11eb-912f-52c4186f8842.png)
That is quite certainly NOT whole Czechia (just [Region Vysočina](https://en.wikipedia.org/wiki/Vyso%C4%8Dina_Region)).

**Comment by tzermias** [comment]
https://v3.wttr.in/Greece shows [Central Greece](https://en.wikipedia.org/wiki/Central_Greece) and [Thessaly](https://en.wikipedia.org/wiki/Central_Greece) regions only.

Remaining regions are available seperately:
- https://v3.wttr.in/Attica 
- https://v3.wttr.in/Peloponnese (Peloponnese, West Greece and Ionian Islands)
- https://v3.wttr.in/Crete
- https://v3.wttr.in/Epirus
- https://v3.wttr.in/Thrace (West Macedonia, Central Macedonia, East Macedonia and Thrace)
- https://v3.wttr.in/Aegean (All aegean islands including Cyclades and Dodecanese)

Detailed info about Greece's regions can be found at [Wikipedia](https://en.wikipedia.org/wiki/Geographic_regions_of_Greece)

**Comment by chubin** [comment]
@mcepl @tzermias 

Thank you for reporting this! First of all, keep in mind that the v3 view is highly experimental,
and it is rather a proof-of-concept than a real implementation. We experiment with it, and gather
feedback and feature requests from the users, and then it will be gradually implemented.

v3 does not support countries names currently, only region names.
I agree, that countries names should be supported too.

That is what must be displayed for Greece and Czechia:

* https://v3.wttr.in/Athens
* https://v3.wttr.in/Prague

What do you think, what should be displayed for location-based quires
in the v3.view? Only small region around the location or the whole country
whith this location highlighted?

For example for *Jihlava*, or for *Larisa*?

**Comment by mcepl** [comment]
Well, it's complicated. Strictly speaking if you want to display something for Jihlava, then for Prague you should display just the region of Prague (https://en.wikipedia.org/wiki/Regions_of_the_Czech_Republic).

**Comment by chubin** [comment]
Yes, exactly; and it is anyway interesting to see the weather in the whole country (if you access v3.wttr.in).

My idea is to make it like this:

1. For queries that correspond to the region names ans countries, display the named region;
2. For locations, display the region or the whole country the location belongs to with the location highlighted.

How to select country or region in the second case:

1. If location is a capital, display the country;
2. If location is not a capital, display country for all countries excluding several exceptions like: USA, Russia, China, India, Canada and maybe some other. Not sure about Brazil, Australia and Argentina.

**Comment by tzermias** [comment]
@chubin Thank you for the prompt resolution!
Greece is now shown correctly!

As for your plan showing corresponding regions, it looks solid. 
Besides, this actually works right now. For example when requesting weather in "Iraklion", Crete region is shown as expected!

**Comment by mystixa** [comment]
Anchorage is listed as a corner case above.. and for that URL I just get a broken image symbol. However I get that for any American city I try for v3, like Portland, Seattle, Chicago, Sacramento all resulted in the same missing image icon.

**Comment by mcepl** [comment]
Ehm, and now https://v3.wttr.in/Czechia shows me just broken image. https://v3.wttr.in/Prague works, https://v3.wttr.in/Jihlava doesn’t.

**Comment by CaptialState** [comment]
I have this error every time i try to access v3.wttr.in

```
Sorry, we are running out of queries to the weather service at the moment.
Here is the weather report for the default city (just to show you what it looks like).
We will get new queries as soon as possible.
You can follow https://twitter.com/igor_chubin for the updates.
======================================================================================
```
